### PR TITLE
Add devtodev to analytics section

### PR DESCRIPTION
**Pull request by Skazochnij**
Comprehensive analytics service that improves your project and saves time for product development

## Project URL
https://www.devtodev.com/

## Description
Add devtodev to analytics section
 
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 3
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 link issue</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
186 | [307](https://httpstatuses.com/307) | https://devtodev.com/ redirects to<br>https://www.devtodev.com/


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by naeemshaikh90** [comment]
@Skazochnij we only supports addition in chronological order (bottom of category).

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 3 link issues</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
1083 | [301](https://httpstatuses.com/301) | https://github.com/MerrickSapsford/Listenable redirects to<br>https://github.com/msaps/Listenable
1210 | [301](https://httpstatuses.com/301) | https://github.com/nestproject/Frank redirects to<br>https://github.com/kylef/Frank
1477 | [301](https://httpstatuses.com/301) | https://github.com/MerrickSapsford/Pageboy redirects to<br>https://github.com/msaps/Pageboy


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -191,6 +191,7 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
 * [MOCA Analytics](https://mocaplatform.com/features) - Paid cross-platform analytics backend.
 * [Countly](https://count.ly) - Open source, mobile & web analytics, crash reports and push notifications platform for iOS & Android.
 * [Abbi](https://github.com/abbiio/iosdk) - A Simple SDK for developers to manage and maximise conversions of all in-app promotions.
+* [devtodev] (https://www.devtodev.com/) - Comprehensive analytics service that improves your project and saves time for product development
```
Comment (dkhamsing) [inline README.md]: There's an extra space after ]

**Comment by dkhamsing** [comment]
@Skazochnij hi, fixing issues with other projects should be done separately. 
Closing

Please resubmit your project
Thanks

**Pull request closed without merging**